test/functional.js

```diff
@@ -563,6 +563,23 @@ module.exports = {
             });
         });
 
+        it('When enabled, TypeScript is compiled!', (done) => {
+            const config = createWebpackConfig('www/build', 'dev');
+            config.setPublicPath('/build');
+            config.addEntry('main', ['./js/render.ts', './js/index.ts']);
+            config.enableTypeScriptLoader();
+
+            testSetup.runWebpack(config, (webpackAssert) => {
+                // check that ts-loader transformed the ts file
+                webpackAssert.assertOutputFileContains(
+                    'main.js',
+                    'document.getElementById(\'wrapper\').innerHTML = "<h1> Hello World!</h1>";'
+                );
```

Comment:
Btw, the test page is currently blank, but for the `vue` loader, I'm adding a `<div id="app"></div>` to it in `setup.js` where `testing.html` is created for the functional tests. You could do the same